apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -191,6 +191,11 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
     // Manage selected dialect
```

Comment:
IMO, these comments are not needed. The var name of the state should infer.

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3566/commits/66c501e3bafdfeba484d242847e688d8d0ee2611